.github/workflows/build_windows.yml

```
@@ -278,14 +278,16 @@ jobs:
-      github.event_name != 'pull_request' &&
-      inputs.framework_ref == '' &&
-      (github.event_name != 'workflow_dispatch' || contains(inputs.platforms, 'windows_portable'))
+    # if: |
+    #   github.event_name != 'pull_request' &&
+    #   inputs.framework_ref == '' &&
+    #   (github.event_name != 'workflow_dispatch' || contains(inputs.platforms, 'windows_portable'))
     runs-on: windows-2025
     steps:
     - name: Clone repository
       uses: actions/checkout@v6
+      submodules: recursive
```

actionlint (1.7.12)

[error] unexpected key "submodules" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)